pkgs/development/python-modules/vdirsyncer/default.nix

```diff
@@ -29,36 +29,22 @@
 
 buildPythonPackage rec {
   pname = "vdirsyncer";
-  version = "0.19.3";
+  version = "0.20.0";
   format = "pyproject";
 
   disabled = pythonOlder "3.7";
```

Comment:
```suggestion
```

noop, we don't have anything older than 3.10.

https://github.com/pimutils/vdirsyncer/blob/8803d5a0865360bc9e69d8bf7c98bc87d7ed8883/pyproject.toml#L17

Reply:
fixed in a commit (under your name) with all your cleanup suggestions. Thank you very much!